tests/test_rag_tools.py

```
@@ -0,0 +1,290 @@
+            delete_data = delete_result.data
+
+            assert "success" in delete_data
+            assert delete_data["success"] is True
+            assert "source_id" in delete_data
+            assert delete_data["source_id"] == source_id
+
+    @pytest.mark.unit
+    async def test_rag_query_parameter_validation(self, mcp_client: Client):
+        """Test RAG query parameter validation."""
+        # Test missing query
+        with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)